integration/spark-common/src/main/scala/org/apache/carbondata/events/Events.scala

```diff
@@ -93,12 +93,20 @@ trait AlterTableAddColumnEventInfo {
 /**
  * event for alter_table_rename
  */
-trait AlterTableCompactionEventInfo {
+trait AlterTableCompactionStatusUpdateEventInfo {
   val carbonTable: CarbonTable
   val carbonLoadModel: CarbonLoadModel
   val mergedLoadName: String
 }
 
+/**
+ * event for alter_table_compaction
+ */
+trait AlterTableCompactionEventInfo {
```

Comment:
Seems no where this is used

Reply:
+case class AlterTableCompactionPostEvent(sparkSession: SparkSession, carbonTable: CarbonTable)
 +  extends Event with AlterTableCompactionEventInfo

Reply:
AlterTableCompactionPostEvent has sparkSession and carbonTable already, why this trait is required